api/v1alpha1/serviceintentions_types.go

```diff
@@ -268,6 +268,9 @@ func (in *ServiceIntentions) Validate() error {
 			}
 		}
 	}
+
+	errs = append(errs, in.validateNamespaces(namespacesEnabled)...)
```

Comment:
Style: I saw both of these styles sprinkled throughout the v1alpha1 package 
```
errs = append(errs, in.validateNamespaces(namespacesEnabled)...)
```
and
```
if err :=  in.validateNamespaces(namespacesEnabled); err != nil {
  errs = append(errs, err...)
}
```

I preferred the one-line and used that in this PR, but it could be a consistency cleanup in a future PR to use the one liner more consistently in the various validate calls instead.

Reply:
Ahh I've never seen that trick. Yes, I think it makes sense to standardize on the one-liner. I've added this to our tech-debt list.